manager.go

```diff
@@ -938,6 +938,14 @@ func (m *Manager) Stop(cleanup MapCleanupType) error {
 	return m.stop(cleanup)
 }
 
+// ForceStop - Detach all eBPF programs and stop perf ring readers, ignore the previous state. The cleanup parameter defines which maps should be closed.
+// See MapCleanupType for mode.
+func (m *Manager) ForceStop(cleanup MapCleanupType) error {
```

Comment:
The only difference between `Stop` and this, is if the state is `elfLoaded` correct? In that case, only `collectionSpec` is set and that isn't touched by `stop()`.